distributed/deploy/tests/test_local.py

```diff
@@ -444,5 +444,17 @@ def test_adapt_then_manual(loop):
                 assert time() < start + 5
 
 
+def test_local_tls(loop):
+    from distributed.utils_test import tls_only_security
+    security = tls_only_security()
+    with LocalCluster(scheduler_port=0, silence_logs=False, security=security,
+                      diagnostics_port=None, ip='tls://0.0.0.0', loop=loop) as c:
+        sync(loop, assert_can_connect_from_everywhere_4_6, c.scheduler.port,
+             protocol='tls',
+             connection_args=security.get_connection_args('client'),
+             timeout=1,
+             )
```

Comment:
Thoughts on adding some trivial assertions here like `assert c.scheduler.address.startswith('tls://')` or is the idea that the `assert_can_connect_from_everywhere` function should handle that for us?

Reply:
that was mostly my thought.  The other one that i should probably add is asserting that non-tls communication does not work,  and i guess more in the way of docs